rosdep/python.yaml

```
@@ -6866,6 +6866,14 @@ python3-lockfile:
   opensuse: [python-lockfile]
   rhel: [python3-lockfile]
   ubuntu: [python3-lockfile]
+python3-luma-core:
+  ubuntu:
+    jammy: [python3-luma.core]
+    noble: [python3-luma.core]
+python3-luma-oled:
+  ubuntu:
+    jammy: [python3-luma.oled]
+    noble: [python3-luma.oled]
```

So I think we should make this whole thing more forwards-compatible, and support Debian:

```suggestion
python3-luma-core:
  debian: [python3-luma.core]
  ubuntu:
    '*': [python3-luma.core]
    focal: null
python3-luma-oled:
  debian: [python3-luma.oled]
  ubuntu:
    '*': python3-luma.oled
    focal: null
```